File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
 
+## Modify Footnote and Endnote content
+
+The following code example shows how to modify the footnote/endnote content from the word document.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Loads the template document
```

Comment:
//Load an existing Word document.
Using (WordDocument document = new WordDocument("Template.docx")😉
{ //Access the footnote from Word document.
WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
//Clear the footnote contents.
footnote.TextBody.ChildEntities.Clear();
//Add new paragraph to footnote
WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
//Set the footnote character format.
footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
//Append footnote text.
footnoteParagraph.AppendText(" Footnote is modified.");
//Access endnote in Word document.
WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
//Clear endnote content.
endnote.TextBody.ChildEntities.Clear();
//Add new paragraph to endnote
WParagraph endnoteParagraph = endnote.TextBody.AddParagraph() as WParagraph;
//Sets the endnote character format.
endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
//Append endnote text.
endnoteParagraph.AppendText(" Endnote is modified.");
//Save the Word document.
document.Save("Sample.docx", FormatType.Docx);
}

Reply:
Done